README.md

```diff
@@ -76,6 +79,169 @@ INFO MKE cluster admin UI: https://test-mke-cluster.example.com
 INFO You can also download the admin client bundle with the following command: launchpad client-config
 ```
 
+### Mirantis Launchpad Provider + Mirantis AWS Module example
+```
+module "provision" {
+  source = "terraform-mirantis-modules/launchpad-aws/mirantis"
+
+  aws_region = var.aws_region
+
+  cluster_name = var.cluster_name
+
+  master_count       = var.master_count
+  master_type        = var.master_type
+  master_volume_size = var.master_volume_size
+
+  worker_count       = var.worker_count
+  worker_type        = var.worker_type
+  worker_volume_size = var.worker_volume_size
+
+  msr_count            = 0
+  windows_worker_count = 0
+}
+
+// launchpad install from provisioned cluster
+resource "launchpad_config" "cluster" {
+  # Tell the launchpad provider to not bother uninstalling
+  # the container products on terraform destroy operations.
+  skip_destroy = true
+
+  metadata {
+    name = var.cluster_name
+  }
+  spec {
+    cluster {
+      prune = true
+    }
+
+    # Here we define hosts to be used in the installation
```

Comment:
I think that we can ignore these long descriptive comment blocks, as they relate directly to an output from this module.